### ISSUE TITLE
0.X recommendation 5: Permalink Issue

### ISSUE THREAD

**Issue by Telstar1980** [iseulde/wp-front-end-editor#138]
Although the permalink does appear, there is no indication that we need to manually edit this in the inline editor. 

The result is that, once published, the posts don't appear as their permalink structure is default rather than selected structure. This either needs to auto-update as it does in the backend editor, or needs indication to writers to edit this part too...

**Comment by RugWarrior** [comment]
Can this be closed then?

**Comment by bryanwillis** [comment]
I just got notified about this because I was linked to another comment I made. Anyway, I don't think we should close this as it's a pretty important factor, in my opinion. We could do something simple using jQuery/javsacript like this...

```javascript
    if (window.location.href.indexOf('?p=') >= 0) {
        $('.fee-title').bind('keyup keypress blur', function () {
            var myStr = $(this).text()
            $('.fee-slug').text(myStr.replace(/(^\s+|[^a-zA-Z0-9 ]+|\s+$)/g, "").replace(/\s+/g, "-").toLowerCase());
        });
    }
```

There's probably a better way to do this with php, but I'd have to look take a look at how front-end-editor is overriding the url. Basically we just have to make sure it is a new post because we don't want to auto change the url if the title is changed later (or maybe we do?). If wanting to use php we could maybe check for a redirect and parse the url or just check the post status if front-end-editor doesn't have a built in function to check if it's new or not:

```php
<?php if ( in_array( $post->post_status, array( 'auto-draft', 'draft', 'pending' ) ) ) { ?>
			
<?php }
```
